tests/unit/test_network_auth.py

```diff
@@ -406,7 +406,7 @@ def __call__(
         stdin: Optional[Any] = None,
         stdout: Optional[Any] = None,
         input: Optional[bytes] = None,
-        check: Optional[bool] = None
+        check: Optional[bool] = None,
```

Comment:
Nit: Unrelated change. I don't *really* care, but if we did have to revert, it might be a nuisance to have this reverted as well.

Reply:
This change _is_ related, see https://github.com/pypa/pip/pull/11909#issuecomment-1493209124.